capture/src/camera.cpp

```diff
@@ -85,13 +87,51 @@ void camera::init_camera(ASI_CAMERA_INFO &CamInfo)
     {
         errx(1, "No cameras connected.");
     }
-    else if (num_devices > 1)
+    else
     {
-        warnx("Found %d cameras, first of these selected.\n", num_devices);
-    }
+        if (cam_name == nullptr) {
+            asi_rtn = ASIGetCameraProperty(&CamInfo, 0);
+            if (asi_rtn != ASI_SUCCESS) {
+                errx(1, "ASIGetCameraProperty error: %s", asi_error_str(asi_rtn));
+            }
+
+            warnx("Found %d cameras; arbitrarily selecting %s.",
+                num_devices, CamInfo.Name);
+        } else {
+            // TODO: check both ASIGetID and ASIGetSerialNumber with our cameras
```

Comment:
`ASIGetID` sounds promising. I like that better than using the model name. Seems almost certain that this ID is meant to solve exactly this problem.